run-tests.php

```diff
@@ -1575,15 +1575,16 @@ function run_test($php, $file, $env)
 
 	// Additional required extensions
 	if (array_key_exists('EXTENSIONS', $section_text)) {
-		$ext_dir=`$php -r 'echo ini_get("extension_dir");'`;
+		$ext_dir=`$php -r "echo ini_get('extension_dir');"`;
```

Comment:
I mainly do development on Ubuntu, and this should work on that platform, since there's no variable expansion in the argument. I can't speak for any other platform. Initially, I tried to use escapeshellarg (which in theory would be the safest cross-platform solution), but according to the documentation:

> On Windows, escapeshellarg() instead replaces percent signs, exclamation marks (delayed variable substitution) and double quotes with spaces and adds double quotes around the string.

Which doesn't work at all because the code then becomes: `echo ini_get(extension_dir);` and `echo implode( , , get_loaded_extensions());`

I will refactor it to a branch if you'd prefer.